### ISSUE TITLE
Ignore closed source vulnerabilities

### ISSUE THREAD

**Issue by sbs2001** [nexB/vulnerablecode#304]
More != Better 

We don't care about vulnerabilities in windows. 

https://git.launchpad.net/ubuntu-cve-tracker/tree/ignored/not-for-us.txt is a good start .

**Comment by pombredanne** [comment]
@sbs2001 agreed at some level, but we want to be careful as proprietary software vulnerabilities may be at times about some FOSS packages that's bundled, so discarding it upfront may not be a good thing at all times.
https://nvd.nist.gov/vuln/detail/CVE-2020-0236 for instance may not be for Ubuntu ... Yet it may be for us? TBD

**Comment by armijnhemel** [comment]
> More != Better
> 
> We don't care about vulnerabilities in windows.

I have problems with this. First of all, equating "windows" with "not open source" is an oversimplification: there is a lot of open source used in Windows (for more than 25 years) and this is only increasing. Also, users of VulnerableCode might actually be interested in this data.
